src/main/java/server/ftpcommands/CommandRegistry.java

```diff
@@ -10,96 +11,111 @@
 
     private CommandResponder commandResponder;
     private FileConnection fileConnection;
+    private NameGenerator nameGenerator;
 
     public CommandRegistry(CommandResponder commandResponder, FileConnection fileConnection) {
         this.commandResponder = commandResponder;
         this.fileConnection = fileConnection;
+        this.nameGenerator = new NameGenerator(fileConnection::isUniqueFileName);
     }
 
     public void STOR(String fileName) {
         try {
-            commandResponder.respond(150, "OK receiving File");
+            sendResponse(150, "OK receiving File");
             fileConnection.store(fileName);
-            commandResponder.respond(250, "OK File stored");
+            sendFormattedResponse(250, "OK %s stored" , fileName);
         } catch (IOException e) {
-            commandResponder.respond(450, "Error storing File");
+            sendResponse(450, "Error storing File");
```

Comment:
Do you think you could refactor that like: `send(Response.OK), ..., send(Response.ERRO)`